packages/server/src/analytics/updateTimeSeries.ts

```diff
@@ -21,20 +21,37 @@ const projectName = processEnvOrThrow('GCLOUD_PROJECT')
 const projectPath = client.projectPath(projectName)
 const baseMetricUrl = 'custom.googleapis.com'
 
+// Not using RAX (Retry Axios) since the request is done through @google-cloud/monitoring
+//
+// Monitoring seems to be throwing a lot of internal errors as our metrics increase in
+// size.
+const retry = async <T>(promise: () => Promise<T>, times = 4): Promise<T> => {
+  for (let i = 0; i < times; i++) {
+    try {
+      return promise()
+    } catch(e) {
+      console.error(e)
+    }
+  }
+
+  throw new Error('Could not complete request')
+}
```

Comment:
Added this on the latest commits, just discovered now that Google Metrics is throwing a lot of 500 errors without explanations of what's causing them--the message says it's completely internal, and that we should try again later.